framework/validators/Validator.php

```diff
@@ -231,9 +231,22 @@ public function init()
     public function validateAttributes($model, $attributes = null)
     {
         if (is_array($attributes)) {
-            $attributes = array_intersect($this->attributes, $attributes);
+            $attrs = [];
```

Comment:
We prefer not to use shorted variables names. Also it makes confusions with `$attributes`